### PR TITLE
You can now build your kernel on BBQLinux too

### PR DESCRIPTION
**Pull request by julianxhokaxhiu**
No special packages need. All are brought by Arch. I've added instructions on how to it.

For users it would be something cool, as no special actions are needed. Install distro, install packages, build, done.

Btw give it a shot, it has much more updated packages (GCC, CCache, etc.) and I'm sure you can obtain more gain using it to compile your Kernel.

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -1,5 +1,25 @@
                        This is Dorimanx Kernel 9.X/8.x
 
+NEW! You can build this kernel easily on BBQLinux (http://www.bbqlinux.org/), an Arch based distro for Android Devs.
+Install it and simply do these commands:
+
+To compile you need:
+$ pacman -S parallel bc
+
+To clone this repo
+$ sudo apt-get install git
```
Comment (StefanescuCristian) [inline README:10]: sudo pacman -S git

File: README [hunk 2]
```diff
@@ -1,5 +1,25 @@
                        This is Dorimanx Kernel 9.X/8.x
 
+NEW! You can build this kernel easily on BBQLinux (http://www.bbqlinux.org/), an Arch based distro for Android Devs.
+Install it and simply do these commands:
+
+To compile you need:
+$ pacman -S parallel bc
```
Comment (StefanescuCristian) [inline README:7]: Don't you also need ccache?
Also, you need to link python to python2. At least linking is needed in pure ArchLinux.

Comment (julianxhokaxhiu) [inline README:7]: This is a pre-made distro for Android development. And has all the thing setup already. The only missing packages are the one used in Dorimanx build_kernel.sh file :)

File: README (hunk 1 above)
Comment (julianxhokaxhiu) [inline README:10]: Already installed in BBQLinux

**Pull request closed without merging**